codebase/superdarn/src.lib/tk/radar.1.22/include/rprm.h

```diff
@@ -33,6 +33,8 @@
 #ifndef _RPRM_H
 #define _RPRM_H
 
+#include "dmap.h"
```

Comment:
Similarly, why is it necessary to include `dmap.h` in this header file?

Reply:
So this one doesn't seem necessary. It probably happened when the make system was not re-compiling my code when I made changes and I started to hunt down any possible error or warning the compiler gave. 

removed it and recommitted.